nip47/event_handler.go

```diff
@@ -369,14 +381,16 @@ func (svc *nip47Service) CreateResponse(initialEvent *nostr.Event, content inter
 	allTags := nostr.Tags{[]string{"p", initialEvent.PubKey}, []string{"e", initialEvent.ID}}
 	allTags = append(allTags, tags...)
 
+	appWalletPubKey, _ := nostr.GetPublicKey(walletPrivKey)
```

Comment:
maybe `walletPrivKey` could be renamed to `appWalletPrivKey` to be consistent with the public one